CHANGELOG.md

```diff
@@ -60,6 +60,10 @@
 * Include the configured `bind_identifier` in `self_binding` violation
   messages.  
   [JP Simard](https://github.com/jpims)
+  
+* Add `library_content_provider` file type to `file_types_order` rule 
+  to allow LibraryContentProvider to be ordered independent from `main_type`
```

Comment:
```suggestion
  to allow `LibraryContentProvider` to be ordered independent from `main_type`.  
```
The two trailing spaces are especially important.

Reply:
Sorry I misread the contribution rules. Fixed.